api_tests/institutions/views/test_institution_users_list.py

```diff
@@ -1,29 +1,42 @@
-from nose.tools import *  # flake8: noqa
-
-from tests.base import ApiTestCase
-from osf_tests.factories import InstitutionFactory, UserFactory
+import pytest
 
 from api.base.settings.defaults import API_BASE
+from osf_tests.factories import (
+    InstitutionFactory,
+    UserFactory,
+)
+
+@pytest.mark.django_db
+class TestInstitutionUsersList:
```

Comment:
@Nesiehr Does adding them as fixtures take longer? I might argue it sets a better precedent if someone later comes to add another test they are unlikely to convert the old test to use fixtures.